php/class-plugin.php

```diff
@@ -304,7 +304,7 @@ public function process_post_query_vars( $post_query_vars ) {
 		);
 
 		// White list allowed meta query compare values.
-		$allowed_meta_query_compare_values = array( '=', '!=', '>', '>=', '<', '<=' );
+		$allowed_meta_query_compare_values = array( '=', '!=', '>', '>=', '<', '<=', 'LIKE' );
```

Comment:
@sayedwp oh, I was thinking that a plugin would add `LIKE`. This could be done via the `customize_object_selector_post_query_vars` filter like this:

```php
add_filter( 'customize_object_selector_post_query_vars', function( $post_query_vars, $original_post_query_vars ) {
    if ( isset( $original_post_query_vars['LIKE'] ) ) {
        $post_query_vars['LIKE'] = $original_post_query_vars['LIKE'];
    }
    return $post_query_vars;
}, 10, 2 );
```

In the end maybe it doesn't matter. The value is still not indexed so all of them will be slow, though `LIKE` would be slower than the other comparisons.